jenkins-pipelines/longevity-twcs-bench-3h.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
```

Comment:
name of file is not compliant with other names.
remove the "bench".
make sure the name is like other names.

Reply:
updated